Description: Replace hostname with a non-existant ip address
 In Ubuntu build environment we do not have DNS resolver, causing an unexpected test
 failure. Use a non-existant IP address instead of hostname.
Author: Vladimir Petko <vladimir.petko@canonical.com>
Forwarded: no
Last-Update: 2023-03-27
--- a/test/jdk/com/sun/jndi/ldap/LdapPoolTimeoutTest.java
+++ b/test/jdk/com/sun/jndi/ldap/LdapPoolTimeoutTest.java
@@ -86,7 +86,9 @@
         env.put("com.sun.jndi.ldap.read.timeout", String.valueOf(READ_MILLIS));
         env.put("com.sun.jndi.ldap.connect.timeout", String.valueOf(CONNECT_MILLIS));
         env.put("com.sun.jndi.ldap.connect.pool", "true");
-        env.put(Context.PROVIDER_URL, "ldap://example.com:1234");
+        // there should not be a LDAP provider on multicast address, but
+        // network stack should be able to send to it
+        env.put(Context.PROVIDER_URL, "ldap://224.0.0.0:1234");
 
         try {
             futures.add(executorService.submit(() -> { attemptConnect(env); return null; }));
